packages/cli/tsconfig.json

```diff
@@ -3,19 +3,17 @@
     "rootDir": "src",
     "outDir": "build",
     "lib": ["ES2020"],
-    "module": "es2022",
+    "module": "NodeNext",
     "target": "es2022",
-    "moduleResolution": "node",
+    "moduleResolution": "NodeNext",
```

Comment:
We need `NodeNext` as @monokle/validation needs package.json `exports` and without it it resolves node_modules incorrectly.